### PR TITLE
Added type definitions for clamscan

### PR DESCRIPTION
**Pull request by vhrechukha**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/clamscan/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        config_file?: string, // Specify config file if it's in an unusual place
+        multiscan?: boolean, // Scan using all available cores! Yay!
+        reloadDb?: boolean, // If true, will re-load the DB on every call (slow)
+        active?: boolean, // If true, this module will consider using the clamdscan binary
+        bypass_test?: boolean, // Check to see if socket is available when applicable
+    };
+    preference?: any; // If clamdscan is found and active, it will be used by default
+}
+
+export default NodeClam;
```
Comment (amcasey) [inline types/clamscan/index.d.ts:198]: It looks like `export = NodeClam` would be closer to the underlying library.

File: types/clamscan/tslint.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
```
Comment (amcasey) [inline types/clamscan/tslint.json:4]: Why was this needed?  Your header looks fine.

File: types/clamscan/tslint.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        "npm-naming": [
+            true,
+            {
+                "mode": "name-only"
+            }
+        ],
+        "trim-file": false
```
Comment (amcasey) [inline types/clamscan/tslint.json:11]: Why was this needed?

File: types/clamscan/tslint.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
+        "npm-naming": [
+            true,
+            {
+                "mode": "name-only"
```
Comment (amcasey) [inline types/clamscan/tslint.json:8]: If this was needed (and it's not clear that it is), please suppress the individual line(s) so that future lint violations aren't ignored.

File: types/clamscan/clamscan-tests.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import NodeClam from 'clamscan';
```
Comment (amcasey) [inline types/clamscan/clamscan-tests.ts:1]: Should this be `import NodeClam = require('clamscan')`?

**Review by amcasey — CHANGES_REQUESTED** [review]

**Review by amcasey — APPROVED** [review]
Thanks!

**Comment by crzdvl** [comment]
Ready to merge
